templates/cointoss/src/skills/toss.ts

```
@@ -0,0 +1,225 @@
+    const walletClient = createWalletClient({
+      account,
+      chain: base,
+      transport: http(),
+    });
+
+    const parsedAmount = BigInt(parseUnits(amountString, 6));
+
+    if (endTime) {
+      const date = new Date(endTime as string);
+      const timestamp = Math.floor(date.getTime() / 1000);
+      if (isNaN(timestamp)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)